docs/examples/kops-test-route53-subdomain.md

```diff
@@ -0,0 +1,870 @@
+|+-------------------------------------------------------+------------+--------------+|
+|||                                 ResourceRecords                                 |||
+||+---------------------------------------------------------------------------------+||
+|||                                      Value                                      |||
+||+---------------------------------------------------------------------------------+||
+|||  ns-1383.awsdns-44.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400  |||
+||+---------------------------------------------------------------------------------+||
+```
+
+Also, do a "dig" test in order to check the zone availability on the Internet:
+
+```bash
+dig +short kopsclustertest.kopeio.org soa
+
+ns-1383.awsdns-44.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400
+
+dig +short kopsclustertest.kopeio.org ns
+
+ns-1383.awsdns-44.org.
+ns-829.awsdns-39.net.
+ns-1973.awsdns-54.co.uk.
+ns-346.awsdns-43.com.
+```
+
+If both your SOA and NS records are there, then your subdomain is ready to be used by KOPS.
+
+
+## AWS/KOPS ENVIRONMENT INFORMATION SETUP:
+
+First, using some scripting and asuming you already configured your "aws" environment on your linux system, use the following commands in order to export your AWS access/secret (this will work if you are using the default profile):
+
+```bash
+export AWS_ACCESS_KEY_ID=`grep aws_access_key_id ~/.aws/credentials|awk '{print $3}'`
+export AWS_SECRET_ACCESS_KEY=`grep aws_secret_access_key ~/.aws/credentials|awk '{print $3}'`
+echo "$AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+```
+
+If you are using multiple profiles (and not the default one), you should use the following command instead in order to export your profile:
+
+```bash
+export AWS_PROFILE=name_of_your_profile
+```
+
+Create a bucket (if you don't already have one) for your cluster state:
+
+```bash
+aws s3api create-bucket --bucket my-kops-s3-bucket-for-cluster-state --region us-east-1
+```
+
+Then export the name of your cluster along with the "S3" URL of your bucket. Add your cluster name to the full subdomain:
+
+```bash
+export NAME=mycluster01.kopsclustertest.kopeio.org
+export KOPS_STATE_STORE=s3://my-kops-s3-bucket-for-cluster-state
+```
+
+Some things to note from here:
+
+- "NAME" will be an environment variable that we'll use from now in order to refer to our cluster name. For this practical exercise, our cluster name is "mycluster01.kopsclustertest.kopeio.org".
+
+
+## KOPS PRIVATE CLUSTER CREATION:
+
+Let's first create our cluster ensuring a multi-master setup with 3 masters in a multi-az setup, two worker nodes also in a multi-az setup, and using both private networking and a bastion server:
+
+```bash
+kops create cluster \
+--cloud=aws \
+--master-zones=us-east-1a,us-east-1b,us-east-1c \
+--zones=us-east-1a,us-east-1b,us-east-1c \
+--node-count=2 \
+--node-size=t2.micro \
+--master-size=t2.micro \
```

Comment:
I have had problems with t2 micros launching. How about you? Make a note that this sizing is not for prod please.

Reply:
You may experience some odd behaviour here with different kops configurations (such as switching from the default networking provider), due to resource limitations. In my tests I found a `t2.medium` being the minimum size I could get a workable cluster with.

Reply:
Ok. I'll add this to the documentation. By the way, the cluster launched and worked OK, but with no deployed apps.